grpc/go-cubic/client.go

```diff
@@ -42,6 +43,8 @@ type Client struct {
 	streamingBufSize uint32
 }
 
+const connectionTimeout = 10*time.Second
```

Comment:
could you please add an option to configure this value in NewClient?

Reply:
Something like this:

// WithDialTimeout returns an Option which sets a timeout value to be used during the gRPC Dial step.
func WithDialTimeout(timeout time.Duration) Option {
	return func(c *Client) error {
		c.dialTimeout = timeout
		return nil
	}
}